uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/ws/JGitFileSystemsEventsManager.java

```diff
@@ -122,6 +122,10 @@ public void close(String name) {
                              ex);
             }
         }
+    }
```

Comment:
@jhrcek the git gots weird, but that is the wrong call:

    public void close(String name) {

        JGitFileSystemWatchServices watchService = fsWatchServices.get(name);

        if (watchService != null) {
            try {
                watchService.close();
            } catch (final Exception ex) {
                LOGGER.error("Can't close watch service [" + toString() + "]",
                             ex);
            }
        }
		
		***** THIS PART HERE was removed
		
        if (jGitEventsBroadcast != null) {
            jGitEventsBroadcast.close();
        }
    }